opal/mca/btl/vader/btl_vader_component.c

```diff
@@ -363,8 +405,13 @@ static int mca_btl_base_vader_modex_send (void)
         modex_size = sizeof (modex.xpmem);
     } else {
 #endif
-        modex_size = opal_shmem_sizeof_shmem_ds (&mca_btl_vader_component.seg_ds);
-        memmove (&modex.seg_ds, &mca_btl_vader_component.seg_ds, modex_size);
+        modex.other.seg_ds_size = opal_shmem_sizeof_shmem_ds (&mca_btl_vader_component.seg_ds);
+        memmove (&modex.other.seg_ds, &mca_btl_vader_component.seg_ds, modex.other.seg_ds_size);
+        modex.other.user_ns_id = mca_btl_vader_parse_proc_ns_user();
+        if (0 == modex.other.user_ns_id) {
+            return OPAL_ERROR;
```

Comment:
Is this really an error?  E.g., are there older versions of Linux that will reach this vader code but not have the namespace link (or have it in an unexpected format) as expected in `mca_btl_vader_parse_proc_ns_user()`?

I.e., is there a case where we will fail to obtain/parse the namespace ID, but that really means there is no namespace, and therefore all processes have the same lack of namespace?

Reply:
Makes sense to not treat '0' as an error. Changed in upcoming force push.